java/ql/src/Security/CWE/CWE-927/SensitiveResultReceiver.qhelp

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+  <overview>
+    <p>If a <code>ResultReceiver</code> is obtained from an untrusted source, such as being unparcelled from an <code>Intent</code> that was received by an exported component, 
+    sensitive data such as passwords should not be sent to it. Otherwise, this sensitive information may be leaked to a malicious application.</p>
```

Comment:
```suggestion
    <p>If a <code>ResultReceiver</code> is obtained from an untrusted source, such as an <code>Intent</code> received by an exported component, 
    do not send it sensitive data. Otherwise, the information may be leaked to a malicious application.</p>
```

Again, please let me know if I've misunderstood, but I believe this can be trimmed a bit for readability